frontend/src/components/Explanation.js

```diff
@@ -1 +1,31 @@
-// This is where the explanation component will be created
\ No newline at end of file
+import React from 'react';
+import { Jumbotron} from 'reactstrap';
+const ExplanationImage = '/images/explanation_background.PNG'
+
+
+//Jumbotron component
+const Explanation = (props) => {
+  return (
+    <div>
+      {/* In the following line the jumbotrons style was inlined. In the style, The background image was set as the image that was previously imported, and its size
```

Comment:
I think it takes longer to read the comments than it does to understand the code. In that case, it may not be helpful for the reader. What do you think?